crates/swc_ecma_compat_es2015/src/arrow.rs

```diff
@@ -57,7 +58,14 @@ use swc_trace_macro::swc_trace;
 /// };
 /// console.log(bob.printFriends());
 /// ```
-pub fn arrow(unresolved_mark: Mark) -> impl Fold + VisitMut + InjectVars {
+pub fn arrow<C>(
```

Comment:
Does this mean that every visitor have to add a generic parameter of type C?

Reply:
Yes, at least all visitors accept `TransformContext`